Gemfile.lock

```diff
@@ -26,6 +29,12 @@ GEM
     parallel (1.22.1)
     parser (3.1.2.1)
       ast (~> 2.4.1)
+    pry (0.14.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    pry-byebug (3.10.1)
```

Comment:
did `Gemfile` change locally for you, Justin, or was the Gemfile.lock just out of date?  I'm ok with pry-byebug in either case;  just want to make sure the code base is consistent with itself.

Reply:
I think the Gemfile.lock was out of date.